src/routes/wallet/p2sh/tx-proposal.routes.js

```diff
@@ -42,7 +43,9 @@ txProposalRouter.post(
           min: 1,
         },
       },
-      toInt: true,
+      customSanitizer: {
+        options: bigIntSanitizer,
+      },
```

Comment:
question(non-blocking): We are going to remove the express-validation dependency correct?

Just checking since we can use zod for validation we could just remove these schemas, but I think this will be in another PR correct?

Reply:
Correct, this is tracked in this issue: https://github.com/HathorNetwork/hathor-wallet-headless/issues/510